### PR TITLE
Enable plugins in functional tests

### PR DESCRIPTION
**Pull request by hemanthnakkina**
Enable dns, telemetry, observability plugins as part of functional tests.

### CODE REVIEW

**Comment by hemanthnakkina** [comment]
Build failed in ceilometer-upgrade. This is a bug where ceilometer-upgrade should be skipped if gnocchi unit is in blocked state.

**Comment by hemanthnakkina** [comment]
> Build failed in ceilometer-upgrade. This is a bug where ceilometer-upgrade should be skipped if gnocchi unit is in blocked state.

Fixed in https://github.com/canonical/snap-openstack/pull/30

**Comment by gboutry** [comment]
Should we disable the plugins at the end of the test ?

File: .github/workflows/build-snap.yml [hunk 1]
```diff
@@ -48,14 +48,36 @@ jobs:
...
           sg snap_daemon "openstack.sunbeam enable loadbalancer"
-          sg snap_daemon "openstack.sunbeam -v configure -a"
-          sg snap_daemon "openstack.sunbeam -v launch"
+          sg snap_daemon "openstack.sunbeam enable dns --nameservers=testing.github."
+          sg snap_daemon "openstack.sunbeam enable telemetry"
+          sg snap_daemon "openstack.sunbeam enable observability"
+          sg snap_daemon "openstack.sunbeam enable vault"
+          sg snap_daemon "openstack.sunbeam enable secrets"
+          sg snap_daemon "openstack.sunbeam enable caas"
+          sg snap_daemon "openstack.sunbeam configure -a"
```
Comment (gboutry) [inline .github/workflows/build-snap.yml:60]: Configure will also run configure caas. You can keep the `sunbeam configure caas`, but it's not needed and will be fast.

Comment (hemanthnakkina) [reply .github/workflows/build-snap.yml:60]: Removed `sunbeam configure caas`

**Pull request closed without merging**